setup.py

```diff
@@ -15,6 +15,7 @@
         'console_scripts': [
             'start-crawl = sh_scrapy.crawl:main',
             'list-spiders = sh_scrapy.crawl:list_spiders',
+            'shub-image-info = sh_scrapy.crawl:shub_image_info',
```

Comment:
I recall one problem with this approach is that it doesn't work when you run the container locally bind mounting the source code directory.